vars/createTestJobPipeline.groovy

```diff
@@ -33,6 +33,15 @@ def call() {
             disableConcurrentBuilds()
             buildDiscarder(logRotator(numToKeepStr: '20'))
         }
+        triggers {
+            parameterizedCron (
+                '''
+                    H 01 * * 0 %branch="scylla-master/releng-testing"
+                    H 01 * * 0 %branch="enterprise"
+                    H 01 * * 0 %branch="scylla-master"
```

Comment:
@fruch I think this can cause a mess in Argus.
Please check together with @k0machi that irrelevant jobs and/or jobs that were moved to different place in Argus aren't showing up as duplicates or in their old location.

Reply:
it's going to be run on master and enterpraise as well.

@roydahan this job only create job if there are missing in their jenkins folder, so if Argus ignored something already created, this won't change anything.

but yes new jobs might appear in Argus, which is a good thing, isn't it ?

Reply:
Argus-Jenkins sync has been automated and happens every 30 minutes, unless this would create any unneeded jobs in the enterprise or master folder no further action is necessary on this front.